CMakeLists.txt

```diff
@@ -6,8 +6,8 @@ project(
   MeshKernel
   VERSION 0.10.0
   DESCRIPTION "Library for creating and editing meshes."
-  LANGUAGES CXX)
-
+  LANGUAGES CXX C)
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
Why is this necessary?

Reply:
bcs triangle is in C, and g++ gnu compiler seems not able to compile .c files. we need to explicitly state is also a c project

Reply:
I meant the "CMAKE_POSITION_INDEPENDENT_CODE ON" part.
Could you please elaborate on that?